### DISCUSSION TITLE
Help - Is there a way to reset copilot?

### DISCUSSION THREAD

**Discussion by Hillsie** [aws/copilot-cli#4933] [General]
How can I reset and start again? 

I was trying copilot and had a stack fail to build. I then tried to delete it and that failed.  So I went and deleted all the elements in AWS cloud console.... everything I could find.  This just made things worse.

Now I am stuck  where I have a limbo initial application with no cloudformation template nor IAM roles or groups. The buckets are also delete. 

When I do a `copilot app show` it defaults to an app that doesn't exist  and I can't seem to create a new one with the same name or init a different one doesn't work like a fresh install.

Advice and suggestions much appreciated. 

My environment is linux

**Comment by Lou1415926** [comment]
@Hillsie Hey there! Copilot stores some SSM parameters to remember the structure and certain configurations for your application. Perhaps you've deleted all the resources but the SSM parameters are still there. You can go ahead and delete these parameters:
```
/copilot/applications/<appname> 
/copilot/applications/<appname>/environments/<envname> 
/copilot/applications/<appname>/components/<svcname> 
```

To make sure you've cleaned up everything you can use `resourcestaggingapi`. With AWS CLI, you can run ⬇️ and find the remaining resources.
```
aws resourcegroupstaggingapi get-resources --tag-filters Key=copilot-application,Values=<the name of t he app>
```
Hope this helps!

**Comment by Hillsie** [comment]
Thanks part one resolved it. And there are only the keys left.
